charts/cluster-api-runtime-extensions-nutanix/templates/helm-repository.yaml

```diff
@@ -45,18 +60,37 @@ spec:
       labels:
         app: helm-repository  # pods have this label, and Service and Deployment depend on it
     spec:
+      initContainers:
+      - name: copy-charts
+        image: "{{ .Values.helmChartsImage.repository }}:{{ default $.Chart.AppVersion .Values.helmChartsImage.tag }}"
+        command: ["/bin/sh", "-c", "cp /charts/*.tar /helm-charts"]
+        imagePullPolicy: "{{ .Values.image.pullPolicy }}"
+        volumeMounts:
+        - name: charts-volume
+          mountPath: "/helm-charts"
       containers:
       - name: helm-repository
         ports:
           - name: serve
             protocol: TCP
             containerPort: 5000
-        image: "{{ .Values.helmRepositoryImage.repository }}:{{ default $.Chart.AppVersion .Values.helmRepositoryImage.tag }}"
-        imagePullPolicy: "{{ .Values.image.pullPolicy }}"
+        image: "{{ .Values.mindthegap.repository }}:{{ .Values.mindthegap.tag }}"
+        imagePullPolicy: "{{ .Values.mindthegap.pullPolicy }}"
```

Comment:
we are not shipping duplicate images. one is an alpine image and the other is mindthegap

Reply:
+1. we could use the same mindthegap image for init-container too. We are not using versioned `alpine` image either. it would be less configuration to just use single image.